.tekton/shellcheck-yaml-lint-pipeline.yaml

```diff
@@ -41,7 +41,7 @@ spec:
         taskSpec:
           steps:
             - name: run-shellcheck
-              image: ghcr.io/openshift-pipelines/test/shellcheck:main
+              image: quay.io/redhat-pipeline-service/shellcheck:latest
```

Comment:
why changing from main to latest?
The CI jobs use the branch name as a tag

Reply:
Yes, 'main' tag exists. But I thought to call using the more standard 'latest' tag. Should not be much of a difference between latest and main right?

Reply:
by default, 3 tags are uploaded with every run - latest, main and commit ID.

Reply:
I'm fine with also having latest pointing to main until we have a v1 release, at which point will need to make latest point to the latest stable release.

Reply:
Currently we neither create release branches nor tags. It has been fine so far but at some point, which is coming close now we will need to do it. Often when dependencies change like the kcp version we need to amend our scripts or manifests.
We are now using kcp 0.6 but we should be able to do work to prepare 0.7 and 0.8 without breaking redeployments based on 0.6.  For each release of Pipeline-Service we should have a clear set of dependency versions: kcp, pac, argo cd, etc
Specifically in regard to main vs latest. At least main says what is inside the tin: the latest build of the main branch. Latest can be anything. We don't have multiple branches and tags currently but when we do latest can be a build from any branch, which may be months behind the main branch (if we happen to support and fix a release branch for months).

Reply:
> For each release of Pipeline-Service we should have a clear set of dependency versions: kcp, pac, argo cd, etc
Yes, agreed. This sort of a dependency matrix could be started once we have the first ever release.

Reg latest vs main : Yes, in the future when we have releases and multiple branches, latest may not necessarily mean latest. But for now, it is. Anyways, let me revert to 'main' to avoid any confusion.